app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3:
+        raise ValueError(
+            "Error: Invalid command format. "
+            "Use 'mv source_path destination_path'."
+        )
+
+    source_path, destination_path = parts[1], parts[2]
+
+    if not os.path.exists(source_path):
+        raise ValueError(
+            f"Error: Source file '{source_path}' not found."
+        )
+
+    if not destination_path:
+        raise ValueError(
+            "Error: Destination path is empty."
+        )
+
+    is_directory = destination_path.endswith("/")
+
+    if is_directory:
+        destination_path = os.path.join(
+            destination_path, os.path.basename(source_path)
+        )
+
+    destination_dir = os.path.dirname(destination_path)
+    if destination_dir and not os.path.exists(destination_dir):
```

Comment:
There is no need to check `os.path.exists(destination_dir)`, presence of `exist_ok=True` will not raise the error dirung the `os.makedirs` operation.